README.md

```diff
@@ -1285,6 +1303,25 @@ parameter to the query:
 +---------------------+
 ```
 
+If the user desires a smaller paging size, this can be accomplished by changing
+the `genquery1.irods_query_limit` configuration setting.  This directly changes
+the irods.query.IRODS_QUERY_LIMIT value and therefore affects Query objects'
+`all` and `get_batches` methods.
+
+Note, however, that expressions such as list(Query(...)) and (row for row in
+Query) are not affected by this setting.
```

Comment:
Not sure about wrapping "(row for row in Query)" in backticks. I take it it's describing a pattern or syntax?
```suggestion
Note, however, that expressions such as `list(Query(...))` and `(row for row in Query)`
are not affected by this setting.
```

Reply:
It's a generator comprehension.